src/Verilator.cpp

```diff
@@ -168,6 +169,8 @@ static void process() {
     // Calculate and check widths, edit tree to TRUNC/EXTRACT any width mismatches
     V3Width::width(v3Global.rootp());
 
+    if (v3Global.useRandomizeMethods()) V3RandomizeMethod::randomizeNetlist(v3Global.rootp());
+
```

Comment:
Please move I think just after coverage insertion as we want it after widthCommit, and also we don't want the randomize stuff itself to be subject to code coverage analysis.